doc/source/api/download_dest.rst

```diff
@@ -1,5 +1,24 @@
 Download destination
-===============================================
+==================================================
+
+.. note::
+   Concrete classes described in this chapter implement methods defined in ``AbstractDownloadDestination``
+
+.. autoclass:: b2sdk.v1.AbstractDownloadDestination
+   :members:
 
 .. autoclass:: b2sdk.v1.DownloadDestLocalFile()
-    :special-members: __init__
+   :no-members:
+   :special-members: __init__
+
+.. autoclass:: b2sdk.v1.DownloadDestBytes
```

Comment:
why no `()` ?

Reply:
Because there are no arguments in the init function so the parentheses are not needed. For other classes putting parentheses will avoid the arguments in the class name in the docs as you can see here https://b2-sdk-python.readthedocs.io/en/master/api/progress.html#b2sdk.v1.TqdmProgressListener